addons/mail/models/res_users.py

```diff
@@ -36,6 +36,11 @@ class ResUsers(models.Model):
     presence_ids = fields.One2many("mail.presence", "user_id", groups="base.group_system")
     # sudo: res.users - can access presence of accessible user
     im_status = fields.Char("IM Status", compute="_compute_im_status", compute_sudo=True)
+    forced_im_status = fields.Selection([
```

Comment:
quick brainstorm on the name: wouldn't `custom_im_status` read better functionally? "force" looks like a very technical term

force also kind of implies "external control" (being forced to do something, ...) whereas here it is just a user choice.

custom Status?
user set status?
selected status?
manual status?
explicit status?
overridden Status?
user-Defined Presence?

Reply:
I'd go with `custom_im_status`, it's the less ambiguous wording and shares the "custom_channel_name" we already allow user to set as a name of conversation.

Reply:
> I think that `custom_im_status` might be misleading, as it could be confused with the concept of a user-defined status message (like a custom status bubble in Discord).

Current `im_status` is technical char "online"/"offline"/etc. so there's no ambiguity right now. The user-text would incite us to name to something like `im_status_message`.

`manual` is fine too, IMO not necessarily better than `custom`.